src/utils/undoDeletions/undoDeletions.ts

```diff
@@ -12,15 +12,15 @@ import { getFileTypeFromPath } from '../getFileTypeFromPath/getFileTypeFromPath'
 */
 
 let highestDeletedOrder = -1;
+const resetDeletedOrder = -1;
```

Comment:
```suggestion
const RESET_DELETED_ORDER = -1;
```
Make sure all caps for CONSTANTS :)